CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Minor: Added dank reasoning for the vanish module
```

Comment:
I don't want to kill the fun but I would appreciate if this was worded in a more "technical" style if you understand me